Add 'save' command for saving images as jpg files.

johnhany opened the pull request:
Certain machine learning tools require every training image files contained in one folder, hence the 'save' command is needed, at least for my work:)

johnhany committed:
Add 'save' command to save images as jpg files.

Add 'save' entry in command choices as well as in help text. The new 'save' command uses opencv interface for writing image files to disk.

johnhany committed:
Add usage for 'save' command

fyu commented:
If only saving images in one folder is concerned here, does "--flat" for export serve the purpose?

johnhany commented:
I tried `export --flat` and did get all the images in one folder with extension *.webp. But I wanted them as *.jpg.
I just find that changing this [line](https://github.com/fyu/lsun/blob/master/data.py#L49) to  `image_out_path = join(image_out_dir, key + '.jpg')` will just meet the purpose.  AND it's a lot faster than `cv2.imwrite`.
If you think it'll be useful, I can commit another pull request with this method.

fyu commented:
I see. cv2.imwrite may be unavoidable to make the generated images supported by general image io. I will commit a fix for this soon.

johnhany commented:
Thank you for that:) BTW, this dataset is fantastic!!!

johnhany committed:
Add save_images_fast() with lmdb port

Change exported file format from *.webp to *.jpg in export_images(), and we get save_images_fast().
It's a lot faster than cv2.imwrite() when exporting jpeg files.
